homeassistant/components/envisalink/__init__.py

```diff
@@ -250,11 +250,12 @@ class EnvisalinkDevice(Entity):
 
     _attr_should_poll = False
 
-    def __init__(self, name, info, controller):
+    def __init__(self, name, info, controller, unique_id):
```

Comment:
It would be good to add type hints here.
```suggestion
    def __init__(self, name: XXX, info: XXX, controller: XXX, unique_id: str) -> None:
```

Reply:
Changed!